inst/startup/cluster_setup.sh

```diff
@@ -1,30 +1,4 @@
 #!/bin/bash
-
-# Entry point for the start task. It will install the docker runtime and pull down the required docker images
-# Usage:
-# setup_node.sh
-
-apt-get -y install linux-image-extra-$(uname -r) linux-image-extra-virtual
```

Comment:
Reason for this being removed? Just unneeded now? I don't see it  moved to code anywhere.